app/sequencescape/sequencescape/api/v2/aliquot.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
 class Sequencescape::Api::V2::Aliquot < Sequencescape::Api::V2::Base # rubocop:todo Style/Documentation
-  # requires shallow path otherwise get a resource not found issue
-  belongs_to :request, shallow_path: true
+  # requires shallow path otherwise get a resource not found issue TODO: where/when do we get this?
+  has_one :request, shallow_path: true
```

Comment:
What was the driver for this change? Feels a bit dangerous!

Reply:
Why does that feel dangerous to you? Stuart and I just found out it does affect the requests you get through the aliquot. With has_one it just gives us id, with belongs_to we get a full object. Why?